test/scalecube-transport/Rsocket.test.js

```diff
@@ -0,0 +1,22 @@
+import { RSocketProvider } from 'src/scalecube-transport/provider/Rsocket';
+
+describe('Rsocket tests', () => {
+  it('Test', async (done) => {
+    const rSocketProvider = new RSocketProvider();
+    await rSocketProvider.connect();
+    const stream = rSocketProvider.request({
+      type: 'requestStream',
+      serviceName: 'greeting',
+      actionName: 'pojo/many',
+      data: { text: 'Some text to be tested' }
+    });
+    stream.subscribe((data) => {
+      console.log('data', data);
```

Comment:
use `expect` not a console 🙂

Reply:
@oarashi For now I use the test only to debug functionality